test/test-content_negotation.py

```diff
@@ -17,9 +17,26 @@ def test_content_negotiation():
     assert isinstance(res, str)
 
 
+def pkgverBds(pkg, minVer, maxVer=None):
```

Comment:
@hseg can you explain what the `Bds` part of this function name refers to?

And you put in a `maxVer` parameter in case it's needed later?

Reply:
Mentioned in a separate comment, adding here in case you missed it:
`Bds`=bounds, fixed it with latest push
And yes, I added an upper bound in case we need to skip on too-new dependencies too. (The semantics I chose are to compare min<=ver<max because while you usually know your lower bound, the upper bound will be due to versions not yet released)

Reply:
okay, thanks